data/org.freedesktop.impl.portal.Screenshot.xml

```diff
@@ -24,6 +24,8 @@
       @short_description: Screenshot portal backend interface
 
       This screenshot portal lets sandboxed applications request a screenshot.
+
+      This documentation describes version 1 of this interface.
```

Comment:
The last comment; is it risky to "start" at 1, when this is the de facto second version?

Reply:
Good point, it's probably more prudent to jump to 2 already